library/common/jni_interface.cc

```diff
@@ -44,20 +44,20 @@ Java_io_envoyproxy_envoymobile_engine_JniLibrary_runEngine(JNIEnv* env,
 
 // AndroidJniLibrary
 
-extern "C" JNIEXPORT jint JNICALL
-Java_io_envoyproxy_envoymobile_engine_AndroidJniLibrary_initialize(JNIEnv* env,
-                                                                   jclass, // class
-                                                                   jobject connectivity_manager) {
-  // See note above about c-ares.
-  return ares_library_init_android(connectivity_manager);
-}
-
-extern "C" JNIEXPORT jboolean JNICALL
-Java_io_envoyproxy_envoymobile_engine_AndroidJniLibrary_isAresInitialized(JNIEnv* env,
-                                                                          jclass // class
-) {
-  return ares_library_android_initialized() == ARES_SUCCESS;
-}
+//extern "C" JNIEXPORT jint JNICALL
+//Java_io_envoyproxy_envoymobile_engine_AndroidJniLibrary_initialize(JNIEnv* env,
+//                                                                   jclass, // class
+//                                                                   jobject connectivity_manager) {
+//  // See note above about c-ares.
+//  return ares_library_init_android(connectivity_manager);
+//}
+//
+//extern "C" JNIEXPORT jboolean JNICALL
+//Java_io_envoyproxy_envoymobile_engine_AndroidJniLibrary_isAresInitialized(JNIEnv* env,
+//                                                                          jclass // class
+//) {
+//  return ares_library_android_initialized() == ARES_SUCCESS;
+//}
```

Comment:
This could be moved to a conditionally-compiled separate file, but I don't think it's strictly-necessary.